package.json

```
@@ -5,10 +5,12 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint --max-warnings 0",
-    "lint:fix": "next lint --max-warnings 0 --fix",
-    "format": "prettier --write --loglevel=warn '*/**/*.{js,jsx,ts,tsx,html,gql,graphql,json}'",
-    "fix": "yarn format && yarn lint:fix",
+    "lint": "yarn lint:eslint & yarn lint:prettier",
```

せっかくスクリプト同期実行の変更を取り込んでいただけたのであれば、こちらも同様に変更いただけると嬉しいです！
```suggestion
    "lint": "yarn lint:eslint && yarn lint:prettier",
```

---

lint は並列に行っても問題ないと考えているのですが、いかがでしょうか？ 
あくまで静的解析であり元ファイルに変更を加えるような処理ではないので

---

なるほど！それはそう！失礼しました！
このままで大丈夫です！